dynamodb/src/main/scala/awscala/dynamodbv2/Table.scala

```diff
@@ -51,6 +53,25 @@ case class Table(
   def put(hashPK: Any, attributes: (String, Any)*)(implicit dynamoDB: DynamoDB): Unit = putItem(hashPK, attributes: _*)
   def put(hashPK: Any, rangePK: Any, attributes: (String, Any)*)(implicit dynamoDB: DynamoDB): Unit = putItem(hashPK, rangePK, attributes: _*)
 
+  def put[E <: AnyRef](hashPK: Any, entity: E)(implicit dynamoDB: DynamoDB, c: ClassTag[E]) = {
```

Comment:
Can you add explicit return types for public methods?

Reply:
Sure, added.